hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/IOzoneManagerLock.java

```diff
@@ -25,46 +25,128 @@
  */
 public interface IOzoneManagerLock {
 
-  OMLockDetails acquireReadLock(OzoneManagerLock.Resource resource,
+  OMLockDetails acquireReadLock(Resource resource,
```

Comment:
Let's also try to reduce such name changes in this PR in order to make it easier for others to review, or put it in the other preceding rename PR instead.